### PR TITLE
Feature/init

### PR DESCRIPTION
**Pull request by parfienczyk**

### CODE REVIEW

File: src/bot.ts [hunk 1]
```diff
@@ -19,23 +24,58 @@ export class MyBot {
...
-      // read from state.
+
       let count = await this.countProperty.get(turnContext);
-      count = count === undefined ? 1 : ++count;
-      await turnContext.sendActivity(`${count}: You said "${turnContext.activity.text}"`);
+
+      const activityText = this.normalizeActivity(turnContext.activity.text);
+
+      switch (activityText) {
+        case 'bluzg':
```
Comment (wtokarzewski) [inline src/bot.ts]: How about if we Use some namespace, with ENG keywords ??

File: src/bot.ts [hunk 2]
```diff
@@ -19,23 +24,58 @@ export class MyBot {
...
+          await turnContext.sendActivity(SAY_INSULT());
+          break;
+
+        case 'help':
+        case 'pomoc':
```
Comment (wtokarzewski) [inline src/bot.ts]: Same as above.

File: src/bot.ts (hunk 1 above)
Comment (parfienczyk) [reply src/bot.ts]: I agree, maybe some dictionary with keys for functionality

File: src/bot.ts [hunk 3]
```diff
@@ -1,41 +1,53 @@
...
       await turnContext.sendActivity(`[${turnContext.activity.type} event detected]`);
     }
+
     // Save state changes
     await this.conversationState.saveChanges(turnContext);
   }
+
+  async getActivity(type: string, TC: TurnContext) {
+    const activity = {
+      [ACTIVITY_9GAG]: () => say9Gag(TC),
```
Comment (mariuszkalinski) [inline src/bot.ts]: [ACTIVITY_9GAG]: say9Gag,

File: src/consts/insults.ts [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import { INSULTS_ARRAY } from './insults_dictionary';
+
+export const GET_RANDOM_ELEMENT = (arrayItems: string[]): string => arrayItems[Math.floor(Math.random() * arrayItems.length)];
```
Comment (mariuszkalinski) [inline src/consts/insults.ts]: add line break to improve readibility

Comment (parfienczyk) [reply src/consts/insults.ts]: Updated

**Pull request merged**